### PR TITLE
added recipe for bitstruct

### PR DESCRIPTION
**Pull request by danielhrisca**
added recipe for bitstruct

### CODE REVIEW

**Comment by danielhrisca** [comment]
@isuruf hey, can you have a look over this recipe? Thank you!

File: recipes/bitstruct/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (ocefpaf) [inline recipes/bitstruct/meta.yaml]: Please remove all the boilerplate comments from the example recipe.

File: recipes/bitstruct/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    - bitstruct
+
+about:
+  home: https://github.com/eerimoq/bitstruct
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Python bit pack/unpack package.'
+  description: |
+    simplejson is a simple, fast, complete, correct and extensible
```
Comment (ocefpaf) [inline recipes/bitstruct/meta.yaml]: Copy-n-pasta :stuck_out_tongue_winking_eye:
Please add the description for bitstruct here.

**Comment by ocefpaf** [comment]
Thanks @danielhrisca!

**Comment by danielhrisca** [comment]
Thank you!

The JSON description was silly indeed

**Comment by ocefpaf** [comment]
> The JSON description was silly indeed

It happens all the time. We should probably drop the example and encourage users to use `conda-skeleton` instead. (But there are a few kinks in conda-skeleton that needs some adjustments before we do that.)

**Pull request merged**